web/src/integTest/java/com/microsoft/applicationinsights/azuretest/Helpers.java

```diff
@@ -0,0 +1,16 @@
+package com.microsoft.applicationinsights.azuretest;
+
+
+/**
+ * Created by moralt on 30/4/2015.
+ */
+public class Helpers {
```

Comment:
I really don't think that you need a Helper class just for sleep, especially when you use it only in one place.